PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -264,47 +268,56 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
+    cutbasedID_Spring15_tight = "egmGsfElectronIDs:cutBasedElectronID-Spring15-25ns-V1-standalone-tight", 
+    cutbasedID_Fall17_V2_veto = "egmGsfElectronIDs:cutBasedElectronID-Fall17-94X-V2-veto",
+    cutbasedID_Fall17_V2_loose = "egmGsfElectronIDs:cutBasedElectronID-Fall17-94X-V2-loose",
+    cutbasedID_Fall17_V2_medium = "egmGsfElectronIDs:cutBasedElectronID-Fall17-94X-V2-medium",
+    cutbasedID_Fall17_V2_tight = "egmGsfElectronIDs:cutBasedElectronID-Fall17-94X-V2-tight",
     
 )
 
 run2_miniAOD_80XLegacy.toModify(slimmedElectronsWithUserData.userFloats,
-    mvaSpring16GP = cms.InputTag("electronMVAValueMapProducer:ElectronMVAEstimatorRun2Spring16GeneralPurposeV1Values"),
-    mvaSpring16HZZ = cms.InputTag("electronMVAValueMapProducer:ElectronMVAEstimatorRun2Spring16HZZV1Values"),
+    mvaSpring16GP = "electronMVAValueMapProducer:ElectronMVAEstimatorRun2Spring16GeneralPurposeV1Values",
+    mvaSpring16HZZ = "electronMVAValueMapProducer:ElectronMVAEstimatorRun2Spring16HZZV1Values",
 )
 
 run2_miniAOD_80XLegacy.toModify(slimmedElectronsWithUserData.userIntFromBools,
-    mvaSpring16GP_WP90 = cms.InputTag("egmGsfElectronIDs:mvaEleID-Spring16-GeneralPurpose-V1-wp90"),
-    mvaSpring16GP_WP80 = cms.InputTag("egmGsfElectronIDs:mvaEleID-Spring16-GeneralPurpose-V1-wp80"),
-    mvaSpring16HZZ_WPL = cms.InputTag("egmGsfElectronIDs:mvaEleID-Spring16-HZZ-V1-wpLoose"),
-    cutbasedID_Sum16_veto = cms.InputTag("egmGsfElectronIDs:cutBasedElectronID-Summer16-80X-V1-veto"),
-    cutbasedID_Sum16_loose = cms.InputTag("egmGsfElectronIDs:cutBasedElectronID-Summer16-80X-V1-loose"),
-    cutbasedID_Sum16_medium = cms.InputTag("egmGsfElectronIDs:cutBasedElectronID-Summer16-80X-V1-medium"),
-    cutbasedID_Sum16_tight = cms.InputTag("egmGsfElectronIDs:cutBasedElectronID-Summer16-80X-V1-tight"),
-    cutbasedID_HLT = cms.InputTag("egmGsfElectronIDs:cutBasedElectronHLTPreselection-Summer16-V1"),
-    cutbasedID_Spring15_veto = cms.InputTag("egmGsfElectronIDs:cutBasedElectronID-Spring15-25ns-V1-standalone-veto"),
-    cutbasedID_Spring15_loose = cms.InputTag("egmGsfElectronIDs:cutBasedElectronID-Spring15-25ns-V1-standalone-loose"),
-    cutbasedID_Spring15_medium = cms.InputTag("egmGsfElectronIDs:cutBasedElectronID-Spring15-25ns-V1-standalone-medium"),
-    cutbasedID_Spring15_tight = cms.InputTag("egmGsfElectronIDs:cutBasedElectronID-Spring15-25ns-V1-standalone-tight"),
+    mvaSpring16GP_WP90 = "egmGsfElectronIDs:mvaEleID-Spring16-GeneralPurpose-V1-wp90",
+    mvaSpring16GP_WP80 = "egmGsfElectronIDs:mvaEleID-Spring16-GeneralPurpose-V1-wp80",
+    mvaSpring16HZZ_WPL = "egmGsfElectronIDs:mvaEleID-Spring16-HZZ-V1-wpLoose",
+    cutbasedID_Sum16_veto = "egmGsfElectronIDs:cutBasedElectronID-Summer16-80X-V1-veto",
+    cutbasedID_Sum16_loose = "egmGsfElectronIDs:cutBasedElectronID-Summer16-80X-V1-loose",
+    cutbasedID_Sum16_medium = ("egmGsfElectronIDs:cutBasedElectronID-Summer16-80X-V1-medium",
```

Comment:
```suggestion
    cutbasedID_Sum16_medium = "egmGsfElectronIDs:cutBasedElectronID-Summer16-80X-V1-medium",
```
@asculac this should fix the compilation error, considering that the log states

```
*** Error compiling 'src/PhysicsTools/NanoAOD/python/electrons_cff.py'...
  File "src/PhysicsTools/NanoAOD/python/electrons_cff.py", line 299
    cutbasedID_Sum16_tight = "egmGsfElectronIDs:cutBasedElectronID-Summer16-80X-V1-tight",
                           ^
SyntaxError: invalid syntax
```